### DISCUSSION TITLE
Unable to clone repo, git says repo owner exceeded monthly LFS storage

### DISCUSSION THREAD

**Discussion by elfin888** [KlingAIResearch/LivePortrait#380] [General]
Hi team! First of all THANK YOU for this INCREDIBLE tool you've made! It's absolutely amazing and SO useful for visual storytelling and controlled image creation. You're brilliant!!! Thank you for making it opensource!

I tried to clone your repo (via git bash) and pull the LFS files, but git told me an error that the account owner of this repo had exceeded their monthly LFS storage.

I really need to get all the files including the LFS ones for my project. 
Can I give you a donation to cover the download cost or is there any other place I can get these files from?

Thanks!

**Comment by FurkanGozukara** [comment]
I had this error so many times and it is just super annoying :(  GitHub really need to change this behaviour
